app/helpers/application_helper.rb

```diff
@@ -1,2 +1,11 @@
 module ApplicationHelper
+  # Returns the full title on a per-page basis.
+  def full_title page_title
+    base_title = t "title"
+    if page_title.empty?
+      base_title
+    else
+      page_title + " | " + base_title
+    end
```

Comment:
dùng toán tử 3 ngôi
so sánh nil?, present?, empty?, blank? giúp anh

Reply:
nil? là dùng để check nil (null)
empty? dùng để check xem array, hash, string,... có empty không? nếu rỗng (empty) thì là true, ngược lại là false
blank? dùng cho bất kỳ object nào. Trả về true nếu object là giá trị là nil, false, empty, ngược lại false
present? dùng cho bất kỳ object nào, để check object có NOT nil, NOT false, NOT empty hay không? Giá trị trả về ngược với blank?